_shared/project/conf/supervisord-dev.conf

```diff
@@ -0,0 +1,43 @@
+[supervisord]
+nodaemon = true
+silent = true
+{% if cookiecutter.get("_directory") == "pyapp" %}
+
+[program:app]
+command=python3 -m {{ cookiecutter.package_name }}
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+{% endif %}
```

Comment:
For `pyapp`'s it doesn't make sense to have a `gunicorn` process by default so I added one that just runs the project's `__main__.py` instead. I do want to have _something_ in `supervisord-dev.conf` by default so that `make dev` has something to run by default.